src/styles/_catalog.scss

```diff
@@ -1,18 +1,18 @@
 .catalog {
   display: grid;
   justify-content: center;
-  grid-template-columns: repeat(1, $card-width);
+  grid-template-columns: $card-width;
   gap: 48px;
 
-  @media (min-width: 488px) and (max-width: 768px) {
+  @media (min-width: 488px) {
     grid-template-columns: repeat(2, $card-width);
   }
 
-  @media (min-width: 769px) and (max-width: 1024px) {
+  @media (min-width: 768px) {
     grid-template-columns: repeat(3, $card-width);
   }
 
-  @media (min-width: 1025px) {
+  @media (min-width: 1024px) {
     grid-template-columns: repeat(4, $card-width);
   }
```

Comment:
You can make a loop to create all media queries. Just create a correctly mapped list to iterate.